src/test/java/com/ucbos/performance/tests/StudentGeneratorTest.java

```diff
@@ -0,0 +1,42 @@
+package com.ucbos.performance.tests;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.ucbos.performance.config.YmlConfigReader;
+import com.ucbos.performance.generators.XMLDataGenerator;
+import com.ucbos.performance.jmeter.LoadTestConstants;
+import com.ucbos.performance.models.NodeValue;
+import com.ucbos.performance.models.YmlNode;
+
+public class StudentGeneratorTest {
+
+    private static Logger logger = Logger.getLogger("StudentGeneratorTest.class.getName()");
+
+    @Before
+    public void setItemDetailsYaml() throws Exception {
+         YmlConfigReader.readYamlConfigurationAssumingListinSameDocument(LoadTestConstants.STUDENT);
+    }
+
+    @Test
+    // @Ignore
+    public void testAutoGenerateItemDetailsUsingFieldMapper() {
+
+        try {
+            XMLDataGenerator doXMLGenerator = new XMLDataGenerator();
+
+            boolean isFileGenerated = doXMLGenerator.generateOrderDataFiles();
+            assertTrue(isFileGenerated);
```

Comment:
Can you pls add an assertion to check whether files were really generated at the configurated results location?

And also assert , if we request for 3 files whether 3 files were generated or not?